File: openapi/code_samples/go/api_v1_passwordless_email_send/post.go
templateType := scalekit.TemplateTypeSignin
response, err := scalekitClient.Passwordless().SendPasswordlessEmail(
  ctx,
  "john.doe@example.com",
  &scalekit.SendPasswordlessOptions{
    Template:         &templateType,
    ExpiresIn:        100,
    MagiclinkAuthUri: "https://yourapp.com/auth/passwordless/callback",
    TemplateVariables: map[string]string{
      "employeeID":   "EMP523",
      "teamName":     "Alpha Team",
      "supportEmail": "support@yourcompany.com",
    },
  },
)
if err != nil {
  // handle error
  return
}
authRequestId := response.AuthRequestId

File: openapi/code_samples/java/api_v1_passwordless_email_send/post.java
TemplateType templateType = TemplateType.SIGNIN;
Map<String, String> templateVariables = new HashMap<>();
templateVariables.put("employeeID", "EMP523");
templateVariables.put("teamName", "Alpha Team");
templateVariables.put("supportEmail", "support@yourcompany.com");

SendPasswordlessOptions options = new SendPasswordlessOptions();
options.setTemplate(templateType);
options.setExpiresIn(100);
options.setMagiclinkAuthUri("https://yourapp.com/auth/passwordless/callback");
options.setTemplateVariables(templateVariables);

SendPasswordlessResponse response = scalekitClient.passwordless().sendPasswordlessEmail(
    "john.doe@example.com",
    options
);

String authRequestId = response.getAuthRequestId();
